lib/frame.js

```diff
@@ -0,0 +1,25 @@
+class Frame {
+    constructor(bowls) {
+        this.bowls = bowls
+        this.MaxScore = 10
+    }
+    score = () => {
+        var score = 0
+        this.bowls.forEach(bowl => {
+            score += bowl
+
+        });
+        return score
+    }
+    isStrike = () => {
+
+    }
+    isGutter = () => {
+        return this.score() == 0
+    }
+    isSpare = () => {
+        return this.score() == this.MaxScore;
+    }
+}
+
+module.exports = Frame
```

Comment:
Love that you are using Node and building on what we learnt last week - jumping in with two feet!